packages/edit-site/src/components/dataviews/filters.js

```diff
@@ -16,8 +16,38 @@ export default function Filters( { fields, view, onChangeView } ) {
 			return;
 		}
 
-		field.filters.forEach( ( f ) => {
-			filters[ f.id ] = { type: f.type };
+		field.filters.forEach( ( filter ) => {
+			let id = field.id;
+			if ( 'string' === typeof filter ) {
```

Comment:
The logic confuses me a bit and I think it's because of trying to introduce some short hand versions for declaring the filters. For example `enumeration`(for author), get's in this `if` block and the one below. Why not have a single way of declaring filters for now?

I might be missing something though.

Reply:
We need to support the following use case: a field can have different filters. For example:

```js
filters: [
  { id: 'author', type: 'enumeration' },
]
```